### DISCUSSION TITLE
Feature request: Icon Only mode for Tag component

### DISCUSSION THREAD

**Discussion by jnoordsij** [primefaces/community#871] [PrimeVue] [answered]
Similarly to how it is possible to have an icon-only Button, it would be nice to be able to have an icon-only Tag. Currently there is some always-present margin on the icon, which means an icon in a Tag without label is not properly centered.

Possibly related: allowing choosing position of icon in Tag component like in Button.

**Comment by tugcekucukoglu** [comment, marked as answer]
It seems there is a buggy situation. Could you create an issue?

**Reply by jnoordsij** [reply]
Thanks for your response! I've created https://github.com/primefaces/primevue/issues/5067 as a follow-up.

**Comment by tugcekucukoglu** [comment]
Thank you!
